images/windows/toolsets/toolset-2019.json

```diff
@@ -423,7 +423,6 @@
         "common_packages": [
             { "name": "7zip.install" },
             { "name": "aria2" },
-            { "name": "azcopy10" },
```

Comment:
DO Not merge like this, removing to allow me to test postgres while is fixed
- https://github.com/chocolatey-community/chocolatey-packages/pull/2612